pkg/kapp/clusterapply/change_set_view.go

```diff
@@ -48,3 +52,36 @@ func (v *ChangeSetView) Print(ui ui.UI) {
 func (v *ChangeSetView) Summary() string {
 	return v.changesView.Summary() // assumes Print was used before
 }
+
+func (v ChangeSetView) PrintCompleteYamlToBeApplied(ui ui.UI) error {
+
+	for _, view := range v.changeViews {
+		opSt := ""
```

Comment:
what is "St" stand for?

Reply:
opSt is operation strategy. St = strategy